### PR TITLE
feat: add translate ptbr

### PR DESCRIPTION
**Pull request by DittrichLucas**
## Status

READY

## Description

Add the Brazilian Portuguese translation and a new folder to group the translations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+<summary>Translations:</summary>
+
+:brazil: [Português Brasileiro](translations/README-ptbr.md)
+
+</details>
+
+<details>
+    <summary>Translations in progress:</summary>
+</details>
+
```
Comment (fityanos) [inline README.md:15]: Better to remove the `Translations` and `Translations in Progress` sections as the translations dir is clear enough.

File: translations/README-ptbr.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+## 🔰 Introdução
```
Comment (fityanos) [inline translations/README-ptbr.md:1]: Rename your file as `README_ptbr.md`

**Review by fityanos — COMMENTED** [review]
Thanks for your contribution @DittrichLucas 

Minor changes needed.

Would love it if you can submit more languages!

File: README.md (hunk 1 above)
Comment (DittrichLucas) [reply README.md]: solved!

File: translations/README-ptbr.md (hunk 2 above)
Comment (DittrichLucas) [reply translations/README-ptbr.md]: solved!

**Comment by fityanos** [comment]
LGTM 🎯

**Pull request merged**